### PR TITLE
Set Up Post Election Epic Test

### PR DESCRIPTION
**Pull request by jranks123**
## What does this change?

We want to test a reactive message to the election result against our control. This implements the tests, one for the US and one for everywhere else (as per our previous tests).

It also varies over the 3 different CTAs, again like the previous tests. 

## What is the value of this and can you measure success?
We will learn if we can beat our control in big moment with a topical ask. 

## Screenshots

screenshot at nov 09 08-04-31
screenshot at nov 09 08-07-33

## Request for comment

@jfsoul @gtrufitt @desbo @philwills 

## Updated screenshots after Sam's changes
screen shot 2016-11-09 at 11 07 03
screen shot 2016-11-09 at 11 07 29

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/experiments/tests/contributions-epic-post-election-copy-test-united-states.js [hunk 1]
```diff
@@ -126,17 +130,38 @@ define([
             mediator.on('contributions-embed:insert', complete);
         };
 
-        var contributeUrlPrefix = 'co_us_epic_footer_';
+        var contributeUrlPrefix = 'co_ukus_epic_footer_';
```
Comment (philwills) [inline static/src/javascripts/projects/common/modules/experiments/tests/contributions-epic-post-election-copy-test-united-states.js]: Why is this ukus?

File: static/src/javascripts/projects/common/modules/experiments/tests/contributions-epic-post-election-copy-test.js [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+             embed,
+             ajax,
+             commercialFeatures,
+             intersection) {
+
+    return function () {
+
+        this.id = 'ContributionsEpicPostElectionCopyTest';
+        this.start = '2016-11-09';
+        this.expiry = '2016-11-12';
```
Comment (katebee) [inline static/src/javascripts/projects/common/modules/experiments/tests/contributions-epic-post-election-copy-test.js]: Will this fail a test? Expiry should be a weekday

**Review by philwills — APPROVED** [review]
All looks good to me.

File: static/src/javascripts/projects/common/modules/experiments/tests/contributions-epic-post-election-copy-test.js (hunk 2 above)
Comment (philwills) [inline static/src/javascripts/projects/common/modules/experiments/tests/contributions-epic-post-election-copy-test.js]: Thanks @katebee, great spot.

**Pull request merged**